internal/handlers/discord/handler.go

```diff
@@ -207,6 +207,62 @@ func (h *Handler) RegisterCommands(s *discordgo.Session, guildID string) error {
 							Description: "Delete one of your characters",
 							Type:        discordgo.ApplicationCommandOptionSubCommand,
 						},
+						{
+							Name:        "equip",
```

Comment:
we do not need these clunky commands. we use list, then show our character sheet